cegs_portal/search/view_models/v1/experiment.py

```diff
@@ -80,3 +80,15 @@ def experiment_facet_values(cls):
         )
         experiment_facet_value_ids = [fv_id[0] for fv_id in experiment_facet_value_ids if fv_id is not None]
         return FacetValue.objects.filter(id__in=experiment_facet_value_ids).select_related("facet")
+
+    @classmethod
+    def analysis_id_search(cls, expr_id: str):
+        queryset = Experiment.objects.filter(accession_id=expr_id)
+        analysis_id = queryset.values_list("analyses__accession_id", flat=True).first()
+        return analysis_id
+
+    @classmethod
+    def all_analysis_id_search(cls, expr_id: str):
+        queryset = Experiment.objects.filter(accession_id=expr_id)
+        analysis_id = queryset.values_list("analyses__accession_id", flat=True)
```

Comment:
These need to be ordered by analysis__accession_id. https://docs.djangoproject.com/en/5.0/ref/models/querysets#django.db.models.query.QuerySet.order_by

Reply:
These need to be ordered from "oldest" (lowest id) to "newest" (largest id). Note that I changed the previous comment from `analysis__accession_id` to `analysis__id`.